src/ComponentFunctions.jl

```diff
@@ -329,52 +329,26 @@ Base.@kwdef struct StaticDelayEdge{T} <: EdgeFunction
             return new{T}(user_f, dim, coupling, sym)
 
         elseif coupling == :fiducial
-            dim % 2 != 0 && error("Fiducial edges are required to have even dim. ",
-                                  "The first dim args are used for src -> dst ",
-                                  "the second for dst -> src coupling.")
+            dim % 2 == 0 ? nothing : error("Fiducial edges are required to have even dim.
+                                            The first dim args are used for src -> dst,
+                                            the second for dst -> src coupling.")
```

Comment:
The old version with `&&` is more idomatic than using the ternary operator i'd say. Also, multi line strings [are possible](https://github.com/JuliaLang/julia/pull/40753) in Julia, but only from `1.7` onwards if I remeber correctly. Therefore, the `"` and `,` solution is better because it does not include all of the whitespace in the literal string